com.rabbitcompany.passky.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=6.0.2
+Name=Passky
+Comment=Passky is a simple, modern, lightweight, open-source and secure password manager.
+Exec=flatpak run com.rabbitcompany.passky
```

Comment:
No. This should be the executable to be run.

Reply:
Hello,

Can you provide me a link to the documentation on how to create a desktop icon for the Electron application packaged with flatpak?

Reply:
I don't know your app nor how to launch it from the command line. So I can't really guess. 

The gist of it is that:
1. the desktop file doesn't know flatpak
2. should follow the standard https://specifications.freedesktop.org/desktop-entry-spec/latest/
3. be managed upstream because this isn't specific to flatpak either and it's a bug to ship any desktop app on Linux without one.

Reply:
Thanks,

It's 1am here, so I will work on that when I wake up.

Thanks again for your help.